examples/jsonplaceholder.graphql

```diff
@@ -23,5 +23,5 @@ type Post {
   userId: Int!
   title: String!
   body: String!
-  user: User @http(path: "/users/{{value.userId}}")
+  user: User @call(query: "user", args: [{key: "id", value: "{{value.userId}}"}])
```

Comment:
This seems like a recursive call that will never end.

Reply:
Why? This is the example provided on the issue.

Reply:
My bad.